Correct broken entries in usb_modeswitch.conf.

Thanks to Stephen Depooter for the accurate bugreport with this patch.

Index: usb-modeswitch/usb_modeswitch.conf
===================================================================
--- usb-modeswitch.orig/usb_modeswitch.conf	2009-06-11 16:57:48.000000000 +0200
+++ usb-modeswitch/usb_modeswitch.conf	2009-06-11 17:36:56.000000000 +0200
@@ -388,7 +388,7 @@
 #
 # Contributor: Hans Kurent, Denis Sutter, Vincent Teoh
 
-;DefaultVendor=  0x12d1;
+;DefaultVendor=  0x12d1
 ;DefaultProduct= 0x1003
 
 # choose one of these:
@@ -401,7 +401,7 @@
 #
 # Contributor: Dale Lane
 
-;DefaultVendor=  0x12d1;
+;DefaultVendor=  0x12d1
 ;DefaultProduct= 0x1001
 
 # choose one of these:
